apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -844,10 +835,11 @@ class Lti13ContextMembership {
       return this.#membershipsBySub[user.lti13_sub] ?? null;
     }
     for (const match of ['uid', 'email']) {
+      if (!(user[match] in this.#membershipsByEmail)) {
+        continue;
+      }
```

Comment:
Same comment as elsewhere: I really really dislike that this lint ruls is preventing us from writing correct, idiomatic code. Maybe a `Map` is really better here.